edge_computing/image-base-install/ibi-understanding-image-based-install.adoc

```diff
@@ -0,0 +1,92 @@
+
+== Overview of an image-based installation and deployment for {sno} clusters
+
+Service providers continue to deploy their infrastructure at the far edge of the network. This presents challenges in low bandwidth, high latency environments, or disconnected environments, in terms of the cost, time, and expertise needed to install and deploy {sno} clusters.
+
+An image-based approach to installing and deploying {sno} clusters at the far edge of the network overcomes these challenges by separating the installation and deployment stages.
+
+Imaged-based installation::
+Using an image-based installation, you can preinstall multiple hosts with {sno} at a central site, such as a service depot or a factory. You can also validate the base configuration for these hosts and leverage the image-based approach to perform reproducible factory installs at scale by using a single container image.
+
+Image-based deployment::
+The preinstalled and validated hosts are shipped to a remote site where they can be rapidly reconfigured and deployed in a matter of minutes by using a configuration image.
+
+.Overview of an image-based installation and deployment for {sno} clusters
+image::../images/711_OpenShift_IBI_Installation_high-level_0624.png[Overview of an image-based installation and deployment]
+
+[NOTE]
+====
+For standalone clusters, you can use the {product-title} installation program to configure a host as part of the cluster deployment process, see _Performing an image-based deployment for standalone {sno} clusters_.
+
+The instructions for preinstalling a host are the same for both standalone and hub-and-spoke architecture clusters. 
+====
+
+=== Image-based installation for {sno} clusters
+Using the {lcao}, you can generate an OCI container image that encapsulates an instance of a {sno} cluster. This image is derived from a dedicated cluster that you can configure with the target {product-title} version.  
+
+You can reference this image in a live installation ISO to consistently preinstall configured and validated instances of {sno} to multiple hosts. This approach enables the preparation of hosts at a central location, for example in a factory or service depot, before shipping the preinstalled hosts to a remote site for rapid reconfiguration and deployment. 
+
+The following is a high-level overview of the image-based installation process:
+
+* Generate an image from a {sno} cluster.
+* Use the {product-title} installation program to embed the seed image URL, and other installation artifacts, in a live installation ISO.
+* Start the host using the live installation ISO to preinstall the host. 
++
+During this process, the {product-title} installation program installs {op-system-first} to the disk, pulls the image you generated, and precaches release container images to the disk.
+
+* When the installation completes, the host is ready to ship to the remote site for rapid reconfiguration and deployment.
+
+=== Imaged-based deployment for {sno} clusters
```

Comment:
I think this section should mention that there are 2 options for deployment:
1. The RHACM one described here
2. The standalone option